apps/dashboard/pages/donations.tsx

```diff
@@ -1,6 +1,6 @@
 import { useEffect, useState } from 'react'
 
-import { useUpdateEffect } from '@chakra-ui/react'
+import { useUpdateEffect, Box, Spacer, Text, Flex } from '@chakra-ui/react'
```

Comment:
Lint error, Spacer is defined but never used

```suggestion
import { useUpdateEffect, Box, Text, Flex } from '@chakra-ui/react'
```